### PR TITLE
Scripts for deployment added

### LINKED ISSUE #23: Deployment per branch
Acceptance criteria
- [ ] When the PR is created, the app is automatically deployed to github pages, with a different __endpoint per branch__.

### PR DESCRIPTION
**Pull request by git-toni**
Closes #23 

Note that the acceptance criteria is not met as described. The flow so far is:

1. One pushes PR
2. One can deploy the app via `yarn deploy:now` (note that the latest local changes will be deployed, not necessarily in sync with the remote PR branch)
3. Once it is deployed one can `yarn deploy:updateURL` and the description of the PR will be updated with the URL latest deployment of the branch.

This is not automated although it could be done so via git hooks. I decided it is better to have manual control of the deployments for starters.
<br/><br/><br/><url>LiveURL: https://kiwicomsmart-faq-jubzanorxo.now.sh
</url>
</url>

### CODE REVIEW

**Comment by git-toni** [comment]
Hello

**Comment by git-toni** [comment]
Aloha

**Comment by git-toni** [comment]
Bona tarda

File: package.json [hunk 1]
```diff
@@ -19,12 +19,14 @@
...
     "eslint": "eslint .",
     "eslint:fix": "eslint . --fix",
+    "deploy:print": "yarn babel-node scripts/deploymentUtils.js createComments",
     "translations:collect": "i18next-scanner 'src/**/*.{js,jsx}'",
     "translations:check": "node scripts/checkTranslations.js"
   },
   "main": "lib/index.js",
   "author": "kiwi.com",
   "devDependencies": {
+    "@octokit/rest": "^15.2.5",
```
Comment (git-toni) [inline package.json]: omg so bad

**Comment by git-toni** [comment]
hola

File: webpack.config.js [hunk 2]
```diff
@@ -47,5 +47,6 @@ module.exports = {
   devServer: {
     contentBase: './dist',
     historyApiFallback: true,
+    disableHostCheck: true, // That solved it
```
Comment (jaroslav-kubicek) [inline webpack.config.js]: I don't get this comment, can you specify what was fixed?

File: package.json [hunk 3]
```diff
@@ -81,6 +85,7 @@
     "@kiwicom/orbit-components": "^0.0.0-rc1",
     "axios": "^0.18.0",
     "babel-runtime": "^6.26.0",
+    "graphql": "0.13.2",
```
Comment (jaroslav-kubicek) [inline package.json]: how is this related to this PR?

File: package.json [hunk 4]
```diff
@@ -19,6 +19,8 @@
     "flow:stop": "flow stop",
     "eslint": "eslint .",
     "eslint:fix": "eslint . --fix",
+    "deploy:now": "now --dotenv",
+    "deploy:updateURL": "yarn babel-node scripts/deploymentUtils.js updateLiveURL",
```
Comment (jaroslav-kubicek) [inline package.json]: Shouldn't these scripts be probably added to Circle CI config?

File: scripts/deploymentUtils.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+// @flow
+const resolve = require('path').resolve;
+
+require('dotenv').config({ path: resolve('.env') });
+
```
Comment (jaroslav-kubicek) [inline scripts/deploymentUtils.js]: why is there new line after every import?

File: package.json (hunk 4 above)
Comment (git-toni) [reply package.json]: I agree that's certainly a good option. I left it non-automated for now, so we can manage deployments manually. If automatic re-deployment is desired on CircleCI right now. We would only need to decide whose `now.sh` account we use to do the deployment?

File: package.json (hunk 3 above)
Comment (git-toni) [reply package.json]: I have no idea why this is hightlighted. AFAIK `graphql` package should have been in `package.json` before this PR was created. :confused:

File: scripts/deploymentUtils.js (hunk 5 above)
Comment (git-toni) [reply scripts/deploymentUtils.js]: Weird styling, fixing.

File: webpack.config.js (hunk 2 above)
Comment (git-toni) [reply webpack.config.js]: I forgot to remove the comment. This bypass flag seems necessary for webpack-dev-server to work properly.

File: package.json (hunk 3 above)
Comment (jaroslav-kubicek) [reply package.json]: no, it's already in devDependencies, this looks more like an error

File: package.json (hunk 4 above)
Comment (jaroslav-kubicek) [reply package.json]: Then I'm missing the point of this PR little bit, especially `deploy:updateURL`  🙂

Comment (jaroslav-kubicek) [reply package.json]: ok, I see the description, I can merge it, but it's still not optimal unless it's part of CI

**Comment by jaroslav-kubicek** [comment]
💡  I would update README as well

File: package.json (hunk 3 above)
Comment (git-toni) [reply package.json]: You're totally right, missed it.

File: package.json (hunk 4 above)
Comment (git-toni) [reply package.json]: Ok! I will use my own now.sh token to deploy via circleCI
